test/ui-test/src/screens/StatusCommunityScreen.py

```diff
@@ -430,3 +432,13 @@ def mark_as_read(self, chatName: str):
             test.fail("Chat is not loaded")
         
         click_obj_by_name(CommunityScreenComponents.MARK_AS_READ_BUTTON.value)
+        
+    def click_sidebar_option(self, community_sidebar_option:str):
+        #TODO Make compatible with other sidebar options
+        if community_sidebar_option=="Manage Community":
+            click_obj_by_name(CommunityScreenComponents.WELCOME_MANAGE_COMMUNITY.value)    
+            
+    def verify_option_exists(self, option:str):
+        if option=="Permissions":
```

Comment:
```suggestion
        if option == "Permissions":
```

Same comment than before, maybe we can create an enum with the possible options instead.